corehq/apps/hqwebapp/utils/webpack.py

```diff
@@ -0,0 +1,21 @@
+import json
+from pathlib import Path
+
+import corehq
+
+WEBPACK_BUILD_DIR = Path(corehq.__file__).resolve().parent.parent / "webpack" / "_build"
+
+
+def get_webpack_manifest(filename=None):
+
+    if not filename:
+        path = WEBPACK_BUILD_DIR / "manifest.json"
+    else:
+        path = WEBPACK_BUILD_DIR / filename
+    if not path.is_file():
+        return None
```

Comment:
It would be nice if this function did not have two different return types (`None` or `dict`) because otherwise every caller must check the returned value to see if an error occurred, and must know what the return value `None` means. What do you think of raising a custom exception rather than returning `None`?

Reply:
understandable, however raising a custom exception means that the error would get raised outside of the `webpack_bundle` filter and then we would have the same issue as before, where the error is raised for management commands, tests, etc. I think that it's acceptable to have this function return `None` or a `dict` for the limited use case it has

Reply:
Can this be changed now that this function is only called in the context of filter execution?

Reply:
updated, btw